client/api/analytics/AnalyticsManager.cs

```diff
@@ -54,16 +54,14 @@ public void Stop()
         public void PushToCache(Target target, FeatureConfig featureConfig, Variation variation)
         {
             var cacheSize = analyticsCache.GetAllElements().Count;
```

Comment:
Any concerns about concurrency here?
I guess worst case is we get this count, as the cache is flushed, and we fail to record this eval.